src/components/10-atoms/input-file/react/story.jsx

```diff
@@ -0,0 +1,114 @@
+/* global document */
+import { storiesOf } from '@storybook/html';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import AXAInputFileReact from './AXAInputFileReact';
+import { withMarkdown } from '../../../../../.storybook/addons/markdown';
+import Readme from '../README.md';
+
+storiesOf('Atoms/Input File/React', module)
+  .addDecorator(withMarkdown(Readme))
+
+  /* Default */
+  .add('InputFile - Default', () => {
```

Comment:
@LucaMele We have this redundancy all over the place. This is imho fine for this pull request, but in the future, we should think about Removing the component name in here, since the wrapping package (`Atoms/Input File/React`) already contains the name.

Reply:
@raphaellueckl @NoahLiechti : agree, fine for the PR and needs to be redefined. Also we have 3 times same style and we should start thinking of making a baseButton.. But not for this PR

Reply:
@raphaellueckl @LucaMele should I make an issue?

Reply:
@NoahLiechti  No,  I guess that's ok.